.github/workflows/wikiman.yml

```diff
@@ -0,0 +1,21 @@
+name: wikiman
+
+on:
+  schedule:
+    - cron:  '6 12 * * *'
+  workflow_dispatch:
+
+jobs:
+  checks:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2.3.4
+    - name: copy pacman.yaml
+      run: cp ./sync/pacman.yaml ./sync/current-pacman.yaml 
+    - name: write GH action token to file
+      run: echo ${{ secrets.GITHUB_TOKEN }} > ./sync/.github
```

Comment:
Doesn't this output the secret to the log?

Reply:
Github should avoid outputting the secret, we will see ****s if it would normally appear in the log.

Reply:
looks like this is true.